### ISSUE TITLE
grafana.ini file is owned by root:root instead of root:grafana

### ISSUE THREAD

**Issue by tjb36** [grafana/grafana#91760]
### What happened?

I create a new Docker container with:
`docker run -d --rm --name grafana_container --user grafana -p 3000:3000 grafana/grafana`

Then enter the container as user "grafana":
` docker exec -it -u grafana grafana_container bash`

and check the permissions of the .ini config file:
```
ls -ltr /etc/grafana/
total 80
drwxrwxrwx    8 grafana  root          4096 Jul 26 15:16 provisioning
-rw-r--r--    1 root     root          2986 Jul 26 15:16 ldap.toml
-rw-r--r--    1 root     root         69902 Jul 26 15:16 grafana.ini
```

The .ini config file is owned by root:root.

### What did you expect to happen?

I expected the .ini file to be owned by root, but also expected the group to be "grafana".
(This is described in various places, e.g [here](https://community.grafana.com/t/are-the-ini-files-for-grafana-configuration-only-used-in-startup-can-be-deleted-post-startup-without-impacting-server/82543/2) and [here](https://community.openhab.org/t/grafana-stopped-working/101636/14)).

### Did this work before?

No

### How do we reproduce it?

1. Create docker container and check permissions of .ini file.

### Environment (with versions)?

Grafana: latest
OS: Debian/Docker
Browser:

### Grafana platform?

Docker

**Comment by mehr74** [comment]
I follow up the steps and you're right. Maybe the following line should be changed.
https://github.com/grafana/grafana/blob/8bb548e17b4b4de66c3b9449501c0eb4f02a3869/Dockerfile#L179C25-L180C35
But I am not sure if the docker images are generater from the `Dockerfile` in the repository.

**Comment by tjb36** [comment]
@hairyhenderson 

Yes. I have a user  `monitoring_user` the host which I am using to run several docker containers (related to monitoring applications), by using the `docker run -u` flag.

I am storing the grafana.ini file on the host and then bind-mounting it into the container. Now, in order to access the file on the host (e.g. for editing or backing up, etc), the monitoring_user needs permissions on the file. 

The way I wanted to achieve this was to create a `grafana` group on the host and add the `monitoring_user` to it. If the grafana.ini file was indeed owned by group `grafana` as expected, then this would work correctly. But because the file is owned by root instead, I would need to give root privileges to monitoring_user on the host (which I don't want to do).

**Comment by hairyhenderson** [comment]
@tjb36 It seems that `grafana.ini` is world-readable - is read permission not enough for your use-case? Editing files within a container is not usually done - if you need to make changes to the config file you should probably build a new image to inject your new config file.
